packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.base.tsx

```diff
@@ -72,7 +72,7 @@ export class TeachingBubbleBase extends BaseComponent<ITeachingBubbleProps, ITea
 
     return (
       <Callout
-        target={targetElement}
+        target={targetElement || target}
```

Comment:
Suggestion: Since `targetElement` is the one being deprecated, should we put `target` first in the condition? Like `target || targetElement`.